src/extendables/User/Minion.ts

```diff
@@ -4,20 +4,21 @@ import { User } from 'discord.js';
 import getActivityOfUser from '../../lib/util/getActivityOfUser';
 import { formatDuration } from '../../util';
 import {
-	MonsterActivityTaskOptions,
-	ClueActivityTaskOptions,
-	CraftingActivityTaskOptions,
 	AgilityActivityTaskOptions,
+	AlchingActivityTaskOptions,
+	BuryingActivityTaskOptions,
+	ClueActivityTaskOptions,
 	CookingActivityTaskOptions,
+	CraftingActivityTaskOptions,
+	FiremakingActivityTaskOptions,
 	FishingActivityTaskOptions,
+	FletchingActivityTaskOptions,
 	MiningActivityTaskOptions,
+	MonsterActivityTaskOptions,
+	OfferingActivityTaskOptions,
 	SmeltingActivityTaskOptions,
 	SmithingActivityTaskOptions,
-	FiremakingActivityTaskOptions,
-	WoodcuttingActivityTaskOptions,
-	OfferingActivityTaskOptions,
-	BuryingActivityTaskOptions,
-	FletchingActivityTaskOptions
+	WoodcuttingActivityTaskOptions
```

Comment:
please undo whatever changes you made to the ordering of these. a reorganization like this is better suited for a standalone PR, and isnt something that should go in a PR like this, it makes reviewing harder and adds lots of diffs.